sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -1026,3 +1030,44 @@ def test_ipp_component_create(self, ipp_registry_client: MLClient, randstr: Call
             from_rest_component.outputs["model_output_ipp"]._intellectual_property
             == command_component.outputs["model_output_ipp"]._intellectual_property
         )
+
+    @pytest.mark.skipif(
+        condition=not is_live(), reason="The request body has a variable parameter version and cannot match the record."
```

Comment:
is this a dev env issue or a by-design behavior? If it's a dev env issue, maybe you can try switching file mode from CRLF to LF.

Reply:
It is by design, because i will create new version: new_version = time.strftime("%Y%m%d%H%M%S", time.gmtime())
The version is a variable parameter.

Reply:
I'm not sure if it works but you can try `version = randstr("new_version")`, which will be a random string but works for recording. randstr is a fixture, you can enable it by using `def test_xxx(self, ..., randstr)`

Reply:
it can work.